decksite/static/js/pd.js

```diff
@@ -162,6 +163,14 @@ PD.showLocalTimes = function () {
         $(this).html(t.tz(moment.tz.guess()).format("dddd h:mma z")).parent(".local").show();
     });
 }
+PD.initBugIcons = function() {
+    $(".card.bugged").each(function() {
+        var element = $("<div class=\"bugicon\"> 🐞 </div>")
+        element.insertAfter($(this));
+        Tipped.create(element, $(this).attr("data-bug-desc"), {maxWidth: "200"});
+    });
+}
```

Comment:
Does doing it in js cause the bug icons to "blink" into the page after the page has already rendered? /bugs/ is a good place to check.

Reply:
With the change of strategy of generating the div with the bug icon in mustache, this doesn't happen, so no problem here.